.github/workflows/zizmor.yml

```diff
@@ -17,16 +17,5 @@ jobs:
         uses: actions/checkout@v4
         with:
           persist-credentials: false
-      - name: Setup Rust
-        uses: actions-rust-lang/setup-rust-toolchain@fb51252c7ba57d633bc668f941da052e410add48 # v1
-      - name: Get zizmor
-        run: cargo install zizmor
       - name: Run zizmor
-        run: zizmor --format sarif . > results.sarif
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Upload SARIF file
-        uses: github/codeql-action/upload-sarif@39edc492dbe16b1465b0cafca41432d857bdb31a # v3
-        with:
-          sarif_file: results.sarif
-          category: zizmor
```

Comment:
So the zizmor action also installs rust and uploads the SARIF file?

Or is that just serviced elsewhere?